src/arche/readers/schema.py

```diff
@@ -42,23 +42,31 @@ def __repr__(self):
 
     def get_enums(self) -> List[str]:
         enums = []
-        for k, v in self.raw["properties"].items():
+        # self.raw["properties"].items() has type:
+        # ItemsView[str, Union[str, bool, int, float, None, list[Any]]]
+        properties = cast(
+            ItemsView[str, Dict[str, Any]], self.raw["properties"].items()
+        )
+        for k, v in properties:
             if "enum" in v.keys():
                 enums.append(k)
         return enums
 
     @staticmethod
     def get_tags(schema: RawSchema) -> TaggedFields:
-        tagged_fields = defaultdict(list)
-        for key, value in schema["properties"].items():
+        tagged_fields: Dict[str, List[str]] = defaultdict(list)
```

Comment:
That's a DefaultDict

Reply:
at the line 64, `Schema.get_field_tags` returns a `Dict`, so there's a incompatible error, that's why I changed to `Dict`

Reply:
The last code casts the returnable to dict, but here it's still DefaultDict.
So we use DefaultDict inside and return casted to `dict()`